### ISSUE TITLE
Anti-NMDA Receptor Encephalitis

### ISSUE THREAD

**Issue by nicolevasilevsky** [monarch-initiative/mondo#151]
_From @jmcmurry on November 16, 2017 22:1_

- MONDO_0016317 limbic encephalitis with NMDA receptor antibodies Should also have a mapping to NCIT_C94853 Anti-NMDA Receptor Encephalitis

In terms of phenotype annotations, this disease is an **excellent** candidate for modeling disease staging and temporality. 

It is thought that this condition is massively underdiagnosed because these folks get referred to psychiatrists instead of to neurologists. Prompt diagnosis and treatment (with immunosupressive drugs and surgical removal of teratoma if applicable) and is life-changing and life-saving.

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3983958/
![image](https://user-images.githubusercontent.com/3658863/32916406-d54a99ee-cad0-11e7-8a38-01ceeeb94b89.png)

Mappings should include:
- HP_0000709 Psychosis
- HP_0000746 Delusions
- HP_0100754 Mania
- HP_0000713 Agitation
- HP_0002360 Sleep disturbance
- HP_0100710 Impulsivity
- HP_0000734 Disinhibition
- HP_0002791 Hypoventilation
- HP_0001250 Seizures
- HP_0001259 Coma

No suitable term in HPO yet:
- Changes in speech
- Disorganization
- Movement abnormalities
- Executive dysfunction

Also a note for the Medical Action Ontology; terms and mappings needed for.
- Surgical removal of tumor
- Immunosupressive drugs

@pnrobinson not sure what you think of those terms?

_Copied from original issue: monarch-initiative/monarch-disease-ontology-RETIRED#419_

**Comment by nicolevasilevsky** [comment]
_From @jmcmurry on November 17, 2017 23:5_

Diagnosis requires lumbar puncture to assess CSF, which of course is extremely invasive. Note that assessing Anti-NMDA titers is easy to do but [not widely available in clia-certified labs](http://www.antinmdafoundation.org/laboratories/). At a minimum,  immunological abnormalities could be assessed, steroids or even more aggressive immunosuppression could be appropriate until more definitive results are available.

**Comment by nicolevasilevsky** [comment]
_From @pnrobinson on November 24, 2017 14:53_

?I agree, Can we develop a file format for the common disease annotations that will include temporality and stages and environment? I will implement the format in our curation tool and then we can start to prototype  this. I think there are ways of a simple extension of current formats that will allow us to develop very powerful algorithms.

-Peter


Peter Robinson?

Professor and Donald A. Roux Chair, Genomics and Computational Biology

The Jackson Laboratory for Genomic Medicine

10 Discovery Drive

Farmington, CT 06032

860.837.2095 t | 860.990.3130 m

peter.robinson@jax.org<mailto:peter.robinson@jax.org>

www.jax.org

Robinson lab: https://robinsongroup.github.io/

New Book: Computational Exome and Genome Analysis<https://www.crcpress.com/Computational-Exome-and-Genome-Analysis/Robinson-Piro-Jager/p/book/9781498775984>

The Jackson Laboratory: Leading the search for tomorrow's cures

________________________________

**Comment by nicolevasilevsky** [comment]
_From @jmcmurry on November 24, 2017 21:50_

Sounds good, Peter. Maybe I'm stating the obvious, but I'll put this out
there anyway:
In terms of the format for common disease staging, we probably need to be
careful about distinguishing temporality (weeks/months) and staging (early,
mid, late disease).
Both characterizations are useful but shouldn't be conflated.
In this particular case for Anti-NMDA Receptor Encephalitis, the disease
progression seems to proceed along similar timeframes as far as we know now.
But if you instead look at a disease like HCV the stages as defined by
viral load and other signs and symptoms are much more consistent than are
the actual time frames. Fulminant TB can take weeks or decades to manifest,
if it does at all.

**Comment by nicolevasilevsky** [comment]
@jmcmurry I read the book "[Brain on Fire](https://www.amazon.com/Brain-Fire-My-Month-Madness/dp/1451621388)", per your recommendation - it was really good, I enjoyed it. 

@jmcmurry and @pnrobinson let me know if I can help with the common disease annotations.

**Comment by nicolevasilevsky** [comment]
_From @pnrobinson on December 4, 2017 15:9_

?Hi Nicole & everybody!

I would suggest that we start in earnest in 2018 -- let's settle on a format and a SOP and see if we can curate some diseases and do something new algorithmically with it. I would also say we should probably concentrate on one area of medicine for a start.

-peter


Peter Robinson

Professor and Donald A. Roux Chair, Genomics and Computational Biology

The Jackson Laboratory for Genomic Medicine

10 Discovery Drive

Farmington, CT 06032

860.837.2095 t | 860.990.3130 m

peter.robinson@jax.org<mailto:peter.robinson@jax.org>

www.jax.org

Robinson lab: https://robinsongroup.github.io/

New Book: Computational Exome and Genome Analysis<https://www.crcpress.com/Computational-Exome-and-Genome-Analysis/Robinson-Piro-Jager/p/book/9781498775984>

The Jackson Laboratory: Leading the search for tomorrow's cures

________________________________

**Comment by nicolevasilevsky** [comment]
_From @cmungall on December 4, 2017 16:35_

> Surgical removal of tumor

We should also add a phenotype annotation to association with tumors (typically ovarian teratoma)  in females? As per the ncit text def

**Comment by nicolevasilevsky** [comment]
_From @cmungall on December 4, 2017 16:49_

Do we not have an SOP for adding to the common annotations using the simpler format? E.g. making a new file in https://github.com/monarch-initiative/hpo-annotation-data/tree/master/common-diseases? Why not lets start with that.

I'm also up for using this as a test case for causal models any time. Assume we'd want to include a model of the glutamate NMDA receptor pathway

**Comment by nicolevasilevsky** [comment]
_From @pnrobinson on December 4, 2017 16:53_

Hi Chris,
are you going to be in Portland next week? I think we can finalize a new format as soon as we have the chance to talk and then we can start. The previous common disease files are great as seeds but they will all need revision.

**Comment by nicolevasilevsky** [comment]
_From @cmungall on December 4, 2017 16:54_

no sorry won't be there

**Comment by cmungall** [comment]
@nicolevasilevsky can you summarize what the action item is and assign the relevant person. Or close the ticket

**Comment by nicolevasilevsky** [comment]
Sure!

Action items:
1) Julie had some new term requests for HPO, I created a ticket: https://github.com/obophenotype/human-phenotype-ontology/issues/3569. @pnrobinson - can you review this ticket, and if these terms look okay, I can add them to HPO.

2) Julie would like to annotate this disease using the new common disease annotation model. As far as I understand, that is not ready yet, right @pnrobinson? But it will be soon, is that correct?

When the new annotation tool is ready, I can create these annotations.

**Comment by pnrobinson** [comment]
This would be a challenging but very useful test case for the new model. Let's touch bases next week to discuss how to start. I think that the new format is ready for a v1 -- if we are willing to work on this one disease using say a google spreadsheet it would be useful to test the ideas for the format. I will start to work on code to implement the format in PhenoteFX;

**Comment by nicolevasilevsky** [comment]
created a new ticket in the HPO annotations repo, closing this here.
